tests/e2e/project/issues.spec.ts

```diff
@@ -55,3 +55,65 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
   await page.getByRole("button", { name: "Comment" }).click();
   await expect(page.getByText("Issue editing failed")).toBeVisible();
 });
+
+test("go through the entire ui issue flow", async ({
+  page,
+  authenticatedPeer,
+}) => {
+  const { rid } = await createProject(authenticatedPeer, "commenting");
+
+  await page.goto(`/seeds/127.0.0.1:8070/${rid}`);
+  await page.getByRole("link", { name: "0 issues" }).click();
+  await page.getByRole("link", { name: "New issue" }).click();
+  await page.getByPlaceholder("Title").fill("This is a title");
+  await page
+    .getByPlaceholder("Write a description")
+    .fill("This is a description");
+
+  await page.getByPlaceholder("Add assignee").fill(authenticatedPeer.nodeId);
+  await page.getByPlaceholder("Add assignee").press("Enter");
+  await expect(
+    page.getByText(
+      `did:key:${authenticatedPeer.nodeId.substring(
+        0,
+        6,
+      )}…${authenticatedPeer.nodeId.slice(-6)}`,
+    ),
+  ).toBeVisible();
+
+  await page.getByPlaceholder("Add tag").fill("bug");
+  await page.getByPlaceholder("Add tag").press("Enter");
+  await expect(page.getByText("bug")).toBeVisible();
+  await page.getByPlaceholder("Add tag").fill("documentation");
+  await page.getByPlaceholder("Add tag").press("Enter");
+  await expect(page.getByText("documentation")).toBeVisible();
+
+  await page.getByRole("button", { name: "Submit" }).click();
+  await expect(page.getByText("This is a title")).toBeVisible();
+  await expect(page.getByText("This is a description")).toBeVisible();
+
+  await page.getByRole("button", { name: "edit" }).first().click();
+  await page.getByPlaceholder("Title").fill("This is a new title");
+  await page.getByRole("button", { name: "save" }).click();
+  await expect(page.getByText("This is a new title")).toBeVisible();
+
+  await page.getByPlaceholder("Leave your comment").fill("This is a comment");
+  await page.getByRole("button", { name: "Comment" }).click();
+  await expect(page.getByText("This is a comment")).toBeVisible();
+
+  await page.getByRole("button", { name: "reply" }).click();
+  await page.getByPlaceholder("Leave your reply").fill("This is a reply");
+  await page.getByRole("button", { name: "Reply", exact: true }).click();
+  await expect(page.getByText("This is a reply")).toBeVisible();
+
+  await page.getByRole("button", { name: "Close issue as solved" }).click();
+  await expect(page.getByText("closed as solved")).toBeVisible();
+
+  await page.getByRole("button", { name: "Reopen issue" }).click();
+  await expect(page.getByText("open", { exact: true })).toBeVisible();
+
+  await page.locator('button[name="stateToggle"]').click();
```

Comment:
Can we use the `getByRole` syntax here?

Reply:
Yeah wasn't able to, I did change the `<button name="stateToggle" ..>` to `<button aria-label="stateToggle" ..>` which allows us to use `getByRole`
Seems like the `name` property in `getByRole` for a `button` isn't a `name` attribute but a label or the content of the button